src/ontodia/data/sparql/sparqlModels.ts

```diff
@@ -44,6 +44,12 @@ export interface ConstructElement {
     object: Field;
 }
 
+export interface ImageLink {
+    inst: Field;
+    type: Field;
+    inst2: Field;
```

Comment:
Please be more specific. What is inst2? If it's image URL, you should call property as such.